src/Optimize/OptimizeBase.h

```diff
@@ -28,6 +28,12 @@ class CostFunctionBase
 public:
   typedef T Return_t;
 
+  #ifdef QMC_COMPLEX
```

Comment:
CostFunctionBase.h

Reply:
@PDoakORNL Could you elaborate what you mean?
I understand the ifdef should be removed. I don't get this file name which doesn't exist.

Reply:
When I go and look for a class declaration, coding standards and sanity suggest the file name match the class. This is not @chrislzhao 's fault but since it is being touched and the review sent me looking for this class declaration...